src/vs/workbench/contrib/extensions/browser/extensionsList.ts

```diff
@@ -118,6 +118,7 @@ export class Renderer implements IPagedRenderer<IExtension, ITemplateData> {
 		const reloadAction = this.instantiationService.createInstance(ReloadAction);
 		const actions = [
 			this.instantiationService.createInstance(ExtensionStatusLabelAction),
+			this.instantiationService.createInstance(MigrateDeprecatedExtension, true),
```

Comment:
Nit: Should it be MigrateDeprecatedExtensionAction?

Reply:
Yes - please review - https://github.com/microsoft/vscode/pull/150612 
Thanks.